src/screens/validators/components/list/components/desktop/index.tsx

```diff
@@ -63,6 +66,10 @@ const Desktop: React.FC<{
     });
   });
 
+  useEffect(() => {
+    resetToTop();
+  }, [props]);
+
```

Comment:
Not props.
You didn't pass the `tabs` prop down to this component. We want to listen to `tabs`, not `props` that's too much. Also move this inside a hook folder

Reply:
Updated.